pyvcloud/vcd/org.py

```diff
@@ -107,6 +107,35 @@ def get_catalog(self, name):
                 return self.client.get_resource(link.href)
         raise Exception('Catalog not found.')
 
+    def update_catalog(self, old_catalog_name, new_catalog_name, description):
+        """
+        Update the name and/or description of a catalog.
+        :param old_catalog_name: (str): The current name of the catalog.
+        :param new_catalog_name: (str): The new name of the catalog.
+        :param description: (str): The new description of the catalog.
+        :return:  A :class:`lxml.objectify.StringElement` object describing the updated catalog.
+        """
+        org = self.client.get_resource(self.href)
+        links = get_links(org,
+                          rel=RelationType.DOWN,
+                          media_type=EntityType.CATALOG.value)
+        for link in links:
+            if old_catalog_name == link.name:
+                catalog = self.client.get_resource(link.href)
+                href = catalog.get('href')
+                admin_href = href.replace('/api/catalog/', '/api/admin/catalog/')
+                catalog = self.client.get_resource(admin_href)
```

Comment:
Probably it's a good idea to highlight that the new catalog object we get as a result of the admin call is not the same as the old catalog object, but rather a admin view of the catalog. I think we should use a new variable say adminViewOfCatalog.